wiki/en/Unregistered-Servers.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: wiki
+title: "Running an Unregistered Server"
+lang: "en"
+permalink: "/wiki/Unregistered-Servers"
+---
+
+{% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
+
+# Running an Unregistered Server
+
+It is highly recommended to test your Server by registering it on one of the built-in Directories **first** so as to narrow down any subsequent problems in unregistered mode.
+
+## Setting up a Server behind a home router
+
+If you set up your server at home, you will probably need to change some settings in your router/firewall:
+
+### Port forwarding
+
+People from outside your home network will not be able to see things inside it. To let external Jamulus clients connect to your server, you need to set up port forwarding in your Router's settings. The exact setup differs for every router. For help see your Router's documentation or [portforward.com](https://portforward.com).
```

Comment:
```suggestion
People from outside your home network cannot see things inside it. To let external Jamulus clients connect to your server, you need to set up port forwarding in your router's settings. The exact setup differs for every router. For help see your router's documentation or [portforward.com](https://portforward.com).
```

Reply:
Before I commit that - why is "Router" capitalised?

Reply:
Router is not capitalized anywhere in that suggested change.

Reply:
Why _was_ _Router_ capitalized? Capitalization here only becomes appropriate in more legal or quasi-legal documents, including our legal declarations to users.